package.json

```diff
@@ -6,21 +6,16 @@
   "repository": "github:octokit/webhooks",
   "license": "MIT",
   "author": "Gregor Martynus (https://github.com/gr2m)",
-  "main": "index.json",
-  "files": [
-    "index.d.ts",
-    "index.json",
-    "schema.d.ts",
-    "schema.json"
-  ],
+  "main": "",
+  "files": [],
```

Comment:
For this to be non-breaking, we need to keep these files.

We should be able to easily do this by just having a `cp` step as part of the release flow, that just copies these files out of the new sub-packages, since the actual contents shouldn't have changed.

Reply:
That would double the size of the final package though, since it would included in the dependencies and locally in the package.

Should we instead make them 4 seperate packages until a breaking release can be made?

Reply:
> since it would included in the dependencies and locally in the package.

Then don't include the three new packages as dependencies 😉